Sources/Confidence/EventSenderStorage.swift

```diff
@@ -3,13 +3,13 @@ import Foundation
 struct EventBatchRequest: Encodable {
```

Comment:
let's remove this EventBatchRequest and send a list of events to the uploader instead, `client_secret` and `send_time` can be attached in the uploader. (client_secret can be sent to the uploader during `init`), we can also remove this additional wrapper type.